### PR TITLE
Add lightbulb icon

### PR DESCRIPTION
**Pull request by ericdvb**

### CODE REVIEW

File: src/lightbulb.svg [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-pineapple" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 250 250" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (hellatan) [inline src/lightbulb.svg]: class name is wrong

Comment (ericdvb) [inline src/lightbulb.svg]: :facepalm:

Comment (ericdvb) [inline src/lightbulb.svg]: updated

**Comment by hellatan** [comment]
published:

> \+ dibs-vg@1.10.0

and verified that the lightbulb was being output in the `dist` folder

**Comment by ericdvb** [comment]
Sweet. Thanks!

**Pull request merged**